app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
Not sure I follow exactly, you are saying because we are removing an object from conceptual existence, you are OK treating this API removal as not a breaking change?